kubernetes-client/src/test/java/io/fabric8/kubernetes/client/internal/UtilsTest.java

```diff
@@ -170,7 +171,7 @@ void interpolateStringWithParametersTest() {
   @Test
   void testGetPluralFromKind() {
     // Given
-    Map<String, Class> pluralToKubernetesResourceMap = new HashMap<>();
+    SortedMap<String, Class> pluralToKubernetesResourceMap = new TreeMap<>();
```

Comment:
This test is strongly calling for a refactor to make it aparameterized test.

We'll need to revisit this in a future PR.